frontend/src/js/components/Store.jsx

```diff
@@ -172,6 +168,9 @@ var Store = React.createClass({
         if (res.err) {
           this.setState({page: 'seats', paymentBegun: false});
         } else {
+          if (res.url[0] === '/') { // when skipping Paytrail
```

Comment:
hmm.. how will this behave if we're running it behind a reverse proxy with url /kauppa or so? I mean, http://speksi.fi/kauppa -> internal redirect to http://localhost:3010/ (likely needs to be http://localhost:3010/kauppa to work well)